tests/ha/conf/dm-master.toml

```diff
@@ -0,0 +1,9 @@
+# Master Configuration.
+
+[[deploy]]
+source-id = "mysql-replica-01"
+dm-worker = "127.0.0.1:8262"
+
+[[deploy]]
+source-id = "mysql-replica-02"
+dm-worker = "127.0.0.1:8263"
```

Comment:
Can we remove these now? (and maybe we need to remove them for other cases.)

Reply:
all removed